src/app.ts

```diff
@@ -2,13 +2,24 @@ import express from "express";
 import dotenv from "dotenv";
 import { connectToDatabase } from "./db";
 import userRoutes from "./routes/users";
+import authRoutes from "./routes/auth";
+import transactionRoutes from "./routes/transaction";
 
 dotenv.config();
 
+if (!process.env.PORT) {
+  throw new Error("missing required environment variable: PORT");
+}
+if (!process.env.MONGO_URI) {
+  throw new Error("missing required environment variable: MONGO_URI");
+}
```

Comment:
what if we add more .env in the future? 
think of a solution where you can check the "required"  .env without adding a new if for each env.